pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -335,6 +370,66 @@ func (p *PriorityQueue) Run() {
 	go wait.Until(p.flushUnschedulablePodsLeftover, 30*time.Second, p.stop)
 }
 
+// runRequeuePlugins calls all registered Requeue() and return plugins returning success.
+func (p *PriorityQueue) runRequeuePlugins(ctx context.Context, pod *v1.Pod, event *framework.ClusterEvent, oldObj, newObj interface{}) sets.Set[string] {
+	var successPlugins sets.Set[string]
+	shouldRecordMetric := rand.Intn(100) < p.pluginMetricsSamplePercent
+	for _, pl := range p.requeuePluginMap[pod.Spec.SchedulerName] {
+		s := p.runRequeuePlugin(ctx, pl, pod, event, oldObj, newObj, shouldRecordMetric)
+		if s.IsSuccess() {
+			successPlugins.Insert(pl.Name())
```

Comment:
```suggestion
			if successPlugins == nil {
				successPlugins = sets.New(pl.Name())
			} else {
				successPlugins.Insert(pl.Name())
			}
```

Inserting into a nil map fails.